src/platforms/common/stm32/swo_manchester_dma.c

```diff
@@ -0,0 +1,570 @@
+/*
+ * This file is a contribution to the Black Magic Debug project.
+ * 
+ * Copyright (c) 2024 Stefan Simek, triaxis s.r.o.
```

Comment:
The copyright notice itself is fine, though please follow the code base style for this. Using an email address is technically optional but highly encouraged so we have some way to contact you later if necessary. Please define that you wrote this file. Including UTF-8 encoded Unicode characters to form a completely correct spelling is fine. Do not include a BOM (Byte Order Mark) at the top of the file in this case.

For example:
```c
 * Copyright (C) 2024 Štefan Šimek, triaxis s.r.o.
 * Written by Štefan Šimek <your@contrib.email.address>
```